enterprise_subsidy/apps/transaction/management/commands/backpopulate_transaction_email_and_title.py

```diff
@@ -0,0 +1,65 @@
+"""
+Management command to backpopulate transaction email and title
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+from openedx_ledger.models import Transaction
+
+from enterprise_subsidy.apps.subsidy.models import Subsidy
+from enterprise_subsidy.apps.transaction.utils import batch_by_pk
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Management command for backpopulating transaction email and title
+
+    ./manage.py backpopulate_transaction_email_and_title
+    """
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.dry_run = False
+
+    def add_arguments(self, parser):
+        """
+        Entry point for subclassed commands to add custom arguments.
+        """
+        parser.add_argument(
+            '--dry-run',
+            action='store_true',
+            dest='dry_run',
+            help=(
+                'If set, no updates will occur; will instead log '
+                'the actions that would have been taken.'
+            ),
+        )
```

Comment:
Consider whether we want to optionally skip internal-only subsidies (or maybe skip those by default).

Reply:
tell me more?

Reply:
We might have test subsidy records for say, performance testing, where there's lots and lots of test transaction records.  We might want the ability to have this command ignore test subsidy records, so we don't have to wait on those being updated.